src/modules/reservations/reservation-app.js

```diff
@@ -0,0 +1,145 @@
+import CarsApi from '../home/cars-api';
+import {
+  setReservedToAPI,
+  getDataFromInvolvementAPI,
+} from './involvementAPI';
+
+const carApi = new CarsApi();
+
+const reservationsPopUp = document.createElement('div');
+const body = document.querySelector('body');
+
+const clearInputs = () => {
+  const userNameInput = document.querySelector(
+    '#userNameInput',
+  );
+
+  const startDateInput = document.querySelector(
+    '#startDateInput',
+  );
+
+  const endDateInput = document.querySelector('#endDateInput');
+
+  userNameInput.value = '';
+  startDateInput.value = '';
+  endDateInput.value = '';
+};
+
+const saveReservationToApi = (e) => {
+  const id = e.target.id.split('-').at(-1);
+  const userNameInput = document.querySelector(
+    '#userNameInput',
+  );
+
+  const startDateInput = document.querySelector(
+    '#startDateInput',
+  );
+
+  const endDateInput = document.querySelector('#endDateInput');
+
+  const dataBody = {
+    item_id: id,
+    username: userNameInput.value,
+    date_start: startDateInput.value,
+    date_end: endDateInput.value,
+  };
+
+  setReservedToAPI(dataBody).then(() => {
+    clearInputs();
+    getDataFromInvolvementAPI(id);
+  });
+};
+
+export default function showReservationModal(e) {
```

Comment:
- [ ] according to the requirements the Project should use ES6, which means you should avoid usage of the function keyword, please use arrow functions instead.